src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,59 @@
+package core.basesyntax;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.DataConverter;
+import core.basesyntax.service.FileReader;
+import core.basesyntax.service.FileWriterService;
+import core.basesyntax.service.FruitService;
+import core.basesyntax.service.ReportService;
+import core.basesyntax.service.impl.DataConverterImpl;
+import core.basesyntax.service.impl.FileReaderImpl;
+import core.basesyntax.service.impl.FileWriterServiceImpl;
+import core.basesyntax.service.impl.FruitServiceImpl;
+import core.basesyntax.service.impl.ReportServiceImpl;
+import core.basesyntax.strategy.OperationStrategy;
+import core.basesyntax.strategy.OperationStrategyImpl;
+import core.basesyntax.strategy.operation.BalanceHandler;
+import core.basesyntax.strategy.operation.OperationHandler;
+import core.basesyntax.strategy.operation.PurchaseHandler;
+import core.basesyntax.strategy.operation.ReturnHandler;
+import core.basesyntax.strategy.operation.SupplyHandler;
+import java.io.File;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Main {
+    private static final String INPUT_FILE_PATH = "src" + File.separator
+            + "main" + File.separator
+            + "resources" + File.separator
+            + "reportToRead.csv";
+    private static final String FILE_REPORT = "src" + File.separator
+            + "main" + File.separator
+            + "resources" + File.separator
+            + "finalReport.csv";
```

Comment:
don't create so many objects by using concatenation
```suggestion
    private static final String INPUT_FILE_PATH = "src/main/resources/reportToRead.csv";
    private static final String FILE_REPORT = "src/main/resources/finalReport.csv";
```